seed/models/columns.py

```diff
@@ -1010,7 +1011,8 @@ def select_col_obj(column_name, table_name, organization_column):
 
                 from_org_col = Column.objects.create(
                     organization=organization,
-                    table_name__in=[None, ''],
+                    table_name='',
```

Comment:
See comment above. And also, this would've thrown an error if it ever was hit since `create` can't take the `__in` kwarg.